omod/src/main/webapp/resources/js/ugandaemr-poc.js

```diff
@@ -202,4 +202,140 @@ function enable_disable(field, class_name_prefix, conditions, input_type) {
             }
         );
     });
+}
+
+
+
+function enable_disable_district_tb_reg(selected_option) {
+
+    var class_name = jq(selected_option).attr("class");
+
+    var length = class_name.length;
+
+    var class_id = parseInt(class_name.substring(length - 1, length))
+
+    var row_id = class_id;
+
+    var disable = true;
+    var requires = false;
+
+    var row_1 = '[class^="DistrictTbReg"][class*="Child1"]';
+
+    var xx = row_1;
+    var selected_value = jq(selected_option).find(":selected").attr("value");
+
+    if (selected_value == "90071") {
+        disable = false;
+        requires = true;
+    }
+
+    switch (row_id) {
+        case 1:
+        xx = row_1;
+        break;
+    }
+
+    jq(xx).each(function () {
+        var group = jq(this);
+
+        if (class_name.indexOf('Child') == -1) {
+            /* group.find("input").attr("value", ""); */
+            group.find("input").attr("required", requires);
+            /* group.find('select').prop("selectedIndex", 0); */
+            group.find("input").attr("disabled", disable);
+            group.find('select').attr("disabled", disable);
+            if (requires == false) {
+                group.find('.required').show();
+            }
+            else if (requires == true) {
+                group.find('.required').hide();
+            }
+
+            if (disable) {
+                /* fade out the fields that are disabled */
+                group.find("input").fadeTo(250, 0.25);
+                group.find("select").fadeTo(250, 0.25);
+            } else {
+                /* remove the fade on the fields */
+                group.find("input").fadeTo(250, 1);
+                group.find("select").fadeTo(250, 1);
+            }
+
+        }
+    });
+}
+
+function getMuacCode(age) {
+
+    jq("#muac-code").find("select").attr("style", "pointer-events: none;");
+
+    jq("#muac-score").find("input[type$='text']").keyup(function() {
+        
+        var age =45;
+
+        var muacScore = jq(this).val();
+
+
```

Comment:
@semujju remove extra lines

Reply:
Its one of those functions you asked me to extract to the global js file. Its purpose is to enable fields based on TB RX.